### PR TITLE
Css.tree

### PR DESCRIPTION
**Pull request by algomezb**
Hi, please review my css tree according to design exposed, thanks

### CODE REVIEW

File: src/components/tree/Tree.tsx [hunk 1]
```diff
@@ -1,70 +1,144 @@
...
-// REFERENCE FORMAT: https://github.com/coreofscience/python-sap/blob/main/src/sap/widget.py#L37-L97
+import CopyImage from '../vectors/CopyImage';
+import StarImgage from '../vectors/StarImage';
 
-import DATA from "./data.json";
+import Reference from './Reference';
+
+import './Tree.css';
+import DATA from './data.json';
+import src from '*.bmp';
```
Comment (odarbelaeze) [inline src/components/tree/Tree.tsx:10]: What does this do?

Comment (algomezb) [reply src/components/tree/Tree.tsx:10]: Hi! How are you? I don't know I think you put that there

Comment (odarbelaeze) [reply src/components/tree/Tree.tsx:10]: I don't remember doing so, mind to delete it ?

Comment (algomezb) [reply src/components/tree/Tree.tsx:10]: No! I already deleted it!

Comment (odarbelaeze) [reply src/components/tree/Tree.tsx]: Nice 👍

**Pull request merged**